app/src/main/java/net/gsantner/markor/format/orgmode/OrgmodeActionButtons.java

```diff
@@ -0,0 +1,61 @@
+
+package net.gsantner.markor.format.orgmode;
+
+import android.content.Context;
+
+import net.gsantner.markor.R;
+import net.gsantner.markor.format.ActionButtonBase;
+import net.gsantner.markor.format.markdown.MarkdownReplacePatternGenerator;
+import net.gsantner.markor.frontend.textview.AutoTextFormatter;
+import net.gsantner.markor.model.Document;
+
+import java.util.Arrays;
+import java.util.List;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.StringRes;
+
+public class OrgmodeActionButtons extends ActionButtonBase {
+
+    public OrgmodeActionButtons(@NonNull Context context, Document document) {
+        super(context, document);
+    }
+
+    @Override
+    public List<ActionItem> getActiveActionList() {
+        final ActionItem[] TMA_ACTIONS = {
+                new ActionItem(R.string.abid_common_checkbox_list, R.drawable.ic_check_box_black_24dp, R.string.check_list),
+                new ActionItem(R.string.abid_common_unordered_list_char, R.drawable.ic_list_black_24dp, R.string.unordered_list),
+                new ActionItem(R.string.abid_common_ordered_list_number, R.drawable.ic_format_list_numbered_black_24dp, R.string.ordered_list),
+                new ActionItem(R.string.abid_common_delete_lines, R.drawable.ic_delete_black_24dp, R.string.delete_lines),
```

Comment:
I am modifying some of these in #2106. Please track those changes and match them as necessary

Reply:
It's ok, I have check the code.